### DISCUSSION TITLE
Restarting passenger-app asks for sudo

### DISCUSSION THREAD

**Discussion by FrosT2k5** [domcloud/domcloud#57] [Q&A] [answered]
Running the command : `passenger-config restart-app /`
Shows up error -
```
*** ERROR: You are not authorized to query the status for this Phusion Passenger(R) instance. Please try again with 'sudo'.
```

I used this command till a few days ago and it worked absolutely fine, is the passenger version upgraded which caused this?
Any alternative for manually restarting apps now?

Thank you

**Comment by willnode** [comment, marked as answer]
Checked. I'm still able to restart apps. It's fine. 

It happens like that because there's nothing to restart, and Passenger thinks you need sudo to restart other things (which isn't). 

Because apps live within passenger doesn't live all the time, it's safe to ignore the message. Your changes will be applied anyway after the next run.

You can try to visit your website and restart it, it will work. Remember, you don't need to do this each time you want to apply changes to your web apps.

**Comment by FrosT2k5** [comment]
Yes it was due to messed up passenger config. Thanks for quick reply
